dd-sdk-android/src/main/kotlin/com/datadog/android/log/internal/LogsFeature.kt

```diff
@@ -30,6 +30,17 @@ internal object LogsFeature {
     internal var clientToken: String = ""
     internal var endpointUrl: String = DatadogEndpoint.LOGS_US
     internal var serviceName: String = DatadogConfig.DEFAULT_SERVICE_NAME
+    internal var envName: String = DatadogConfig.DEFAULT_ENV_NAME
```

Comment:
this logic is a bit weird for me...the way I see it the envName is being referenced only to build the envTag when is being set. Maybe we can simplify this by only keeping the envTag and assign it in the constructor : engTag = resolveEnvTag(config.envName). This way you can reuse this utility function also on the other feature.

Reply:
Well the thing is that the envName is also set from the tests,

Reply:
Oh I see. Maybe at least maybe you can extract that function you are using there for CrashFeature and LogsFeature and re - use it.